demo/birds_skip_thought_demo.py

```diff
@@ -12,9 +12,9 @@
 sys.path.append('misc')
 sys.path.append('stageII')
 
```

Comment:
```suggestion

import skipthoughts
```
Otherwise will get a `NameError: name 'skipthoughts' is not defined` error in line 174